.githooks/commit-msg

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+START_LINE=`head -n1 $1`
+PATTERN="[^a-z].*"
+if ! [[ "$START_LINE" =~ $PATTERN ]]; then
+  echo "Bad commit message, Please capitalize the first letter of the commit message"
```

Comment:
This is nice but maybe should have added a few more notes:
- Use imperative style for the commit message
- To read our CONTRIBUTING guide